ui/hooks/useGasFeeInputs.js

```diff
@@ -78,6 +81,29 @@ function getGasFeeEstimate(
   return '0';
 }
 
+/**
+ * This method tries to determine if any estimate level matches the
+ * current maxFeePerGas and maxPriorityFeePerGas values. If we find
+ * a match, we can pre-select a radio button in the RadioGroup
+ */
+function getMatchingEstimateFromGasFees(
+  gasFeeEstimates,
+  maxFeePerGas,
+  maxPriorityFeePerGas,
+) {
+  if (process.env.SHOW_EIP_1559_UI) {
+    return (
+      findKey(
+        gasFeeEstimates,
+        (estimate) =>
+          estimate?.suggestedMaxPriorityFeePerGas === maxPriorityFeePerGas &&
+          estimate?.suggestedMaxFeePerGas === maxFeePerGas,
+      ) || null
+    );
+  }
```

Comment:
@darkwing Can you add the case for legacy low, medium and high estimates? This would require passing in `gasPrice` and checking when its matches the low, medium or high gasPrice in an else condition. (Even after the mainnet hardfork, this might be valuable on some custom networks, if we do provide a gas price estimate api for those.)

Reply:
Done!